src/sysc_3303_project/scheduler_subsystem/ElevatorFaultDetector.java

```diff
@@ -17,18 +17,20 @@
  *
  */
 public class ElevatorFaultDetector {
-	
-	private class ElevatorMoveTimer {
+	protected class ElevatorMoveTimer {
 		public Timer timer;
 		public int elevatorId;
-		
 		private ElevatorMoveTimer(Timer timer, int elevatorId) {
 			this.timer = timer;
 			this.elevatorId = elevatorId;
 		}
+
+		public int getElevatorId() {
+			return elevatorId;
+		}
 	}
-	
-	private List<ElevatorMoveTimer> elevatorMoveTimers;
```

Comment:
If we want accessability in the testing can we change this to "public" instead?

Reply:
Yeah okay ill change that once im back home from work